src/components/molecules/ImageCarousel/ImageCarousel.tsx

```diff
@@ -20,90 +22,96 @@ const ImageCarousel = ({ images }) => {
           </div>
-          <div className={styles.infoSubmit}>
-            <div><h3>Prize Pool</h3></div>
-            <div className={styles.buttonJoin}>
-              <p>Boost the prize pool!</p>
-              <button>JOIN Now</button>
+        </div>
+
+        <div className={styles.showCard}>
+          <div className={styles.cardInfo}>
+            <div className={styles.infoTile}>
+              <p>Objective</p>
+              <h3>Highest Total Kills</h3>
+            </div>
+            <div className={styles.infoSubtitle}>
+              <p>Games Counted (after joining)</p>
+              <p>Up To Best 100 Winning Games</p>
+            </div>
+            <div className={styles.infoDescription}>
+              <p>Queue</p>
+              <p>ARAM</p>
+            </div>
+            <div className={styles.infoPoints}>
+              <p>Owner</p>
+              <p>Pixel Forge Studio</p>
+            </div>
+            <div className={styles.infoSubmit}>
+              <div><h3>Prize Pool</h3></div>
+              <div className={styles.buttonJoin}>
+                <p>Boost the prize pool!</p>
+                <button>JOIN Now</button>
+              </div>
+              <div><p>Entry <span>FREE</span></p></div>
             </div>
-            <div><p>Entry <span>FREE</span></p></div>
           </div>
         </div>
-      </div>
 
-    </div>
+      </div>
 
-    <div className={styles.gameDescription}>
-      <h1>About City Tank</h1>
-      <p>
-        ​Inject new Legendary Lords, Heroes, and Units into your games to vastly expand your methods of play on the campaign map and within battles. The Changeling, Yuan Bo, and Mother Ostankya each bring their own unique campaign mechanics to their races, ​ ​with goals separate from the Ursus storyline in the Realm of Chaos campaign, and new units to help them pursue victory on their terms. 
-        3 new Legendary Lords emerge from the shadows.
-        Enact malign schemes and trickery with the Changeling of Tzeentch.
-        Establish and execute grand stratagems on a global scale as Yuan Bo of Grand Cathay.
-        Conjure powerful hexes and incantations whilst engaging in witchcraft, flipping campaigns on their head as Mother Ostankya of Kislev. 
-        Scatter magical spells far and wide with the Blue Scribes of Tzeentch; a new Legendary Hero. 
-        ​Expand your roster with 12 additional overall units, and enhance your troops with 9 new Regiments of Renown.
-      </p>
+      <div className={styles.gameDescription}>
+        <h1>About City Tank</h1>
+        <p>
+          ​Inject new Legendary Lords, Heroes, and Units into your games to vastly expand your methods of play on the campaign map and within battles. The Changeling, Yuan Bo, and Mother Ostankya each bring their own unique campaign mechanics to their races, ​ ​with goals separate from the Ursus storyline in the Realm of Chaos campaign, and new units to help them pursue victory on their terms. 
```

Comment:
This content should be inside the constant of the specific game instead here for all

Reply:
is adjusted in the next PR